routes/tests/shopifyApiProxy.test.js

```diff
@@ -0,0 +1,102 @@
+const express = require('express');
+const http = require('http');
+const fetch = require.requireActual('node-fetch');
+const fetchMock = require.requireMock('node-fetch');
+
+const shopifyApiProxy = require('../shopifyApiProxy');
+
+const { DISALLOWED_URLS } = shopifyApiProxy;
+const PORT = 3000;
+const BASE_URL = `http://localhost:${PORT}`;
+const API_ROUTE = '/api';
+
+jest.mock('node-fetch');
+
+let session;
+let server;
+describe('shopifyApiProxy', async () => {
+  beforeEach(async () => {
+    fetchMock.mockImplementation(() => ({ status: 200, text: () => Promise.resolve(':)') }));
```

Comment:
is this supposed to be returning a smiley face? At first glance I thought this might be a fixture for un-parsable JSON. Perhaps we should consider either naming the response data to make this clear or picking a simpler default response

Reply:
It doesn't actually matter what it is, since we just need to check that it was proxied correctly. We could put some real data in a fixture and use that or something if we wanted, or just return an empty string, or a different emoji.

Reply:
I guess just `Promise.resolve()` would be ok here since we don't need to use this value directly, and in the test where we check that the body is forwarded we override the mockImplementation anyway.